src/mono/sample/wasm/browser-bench/frame-main.js

```diff
@@ -31,14 +31,17 @@ try {
     }
 
     const runtime = await dotnet
+        .withConfig({
+            maxParallelDownloads: 10000,
```

Comment:
Is it appropriate to do this if most users won't? Shouldn't we measure the default configuration?

Reply:
this is the same as in blazor for Net8, so doing it here should be fine. See https://github.com/dotnet/runtime/issues/89631#issuecomment-1691207621

For Net9 we could make throttling to trigger only after first fail (bad network or lack of browser resources) and then re-enable it on both places.